pkg/core/resources/model/rest/v1alpha1/resource_test.go

```diff
@@ -98,7 +89,7 @@ var _ = Describe("Rest Resource", func() {
 				// and
 				expected := `
 {
-  "type": "CircuitBreaker",
+  "type": "MeshTrafficPermission",
```

Comment:
Instead of switching test cases from one type to another. Should you just add new test cases so you have a test case for old style and a test case for new style?

Reply:
It was a wrong test because `v1alpha1.Resource` is not supposed to be used with the old policy CircuitBreaker. There is a test for CircuitBreaker in the `unversioned.Resource` package